client/asset/eth/multirpc.go

```diff
@@ -59,7 +59,7 @@ const (
 	// HTTPS, but not for other requests.
 	// TODO: Keep a file mapping provider URL to retrieved chain IDs, and skip
 	// the eth_chainId request after verified for the first time?
-	defaultRequestTimeout = time.Second * 20
+	defaultRequestTimeout = time.Second * 10
```

Comment:
I had set this to 20 seconds a while back, but I've come to realize that I had a misconfiguration on my system that brought some glibc bug to the surface that was causing hella long DNS lookup times in some cases.